app/routes/books.rb

```diff
@@ -14,3 +14,9 @@
 post '/books' do
   upsert! Bibliotheca::Book, Bibliotheca::Collection::Books
 end
+
+post '/book/:organization/:repository/fork' do
+  authorize! :writer
+  destination = Mumukit::Auth::Slug.new json_body['organization'], params[:repository]
+  Bibliotheca::Collection::Books.find_by_slug!(slug.to_s).fork_to! destination, bot
+end
```

Comment:
Maybe we extract the logic to a helper to avoid repeating this logic in 3 endpoints